### DISCUSSION TITLE
Copy text formating

### DISCUSSION THREAD

**Discussion by puffitos** [TriliumNext/Trilium#1616] [Q&A] [answered]
Hi all,

just started using trilium today and I am really impressed by its big array of possibilities. I am a CS student and I was using OneNote to keep all my notes, but I've finally seen the wrongness of my "proprietary" ways.

A thing I used to do all the time in OneNote was quickly being able to copy the text formatting (Ctrl+Shift+C) and applying to other parts of my text (like color, text size and so on). Is there a way to do this in trilium? 

Thanks for reading 👍

**Comment by zadam** [comment, marked as answer]
Hi, this would be a CKEditor feature and does not seem currently supported.

https://github.com/ckeditor/ckeditor5/issues/1901 is tracking this in CK repo.

**Reply by puffitos** [reply]
Thanks for the quick reply! I guess I'll just have to wait 😊
